app/router.go

```diff
@@ -188,3 +195,31 @@ type optionValue struct {
 }
 
 func nop(r render.Renderer) render.Renderer { return r }
+
+func addKubernetesTopologies() {
```

Comment:
I wonder weather instead of adding / removing the topology, it would be better to have a flag like "hide if empty" that the UI could honor?